src/scripts/content/turtle.js

```diff
@@ -0,0 +1,58 @@
+/*jslint indent: 2 */
+/*global $: false, document: false, togglbutton: false*/
+
+'use strict';
+
+const join = (elements) => {
+  return Array.from(elements).map(item => item.textContent).join('')
+};
+
+const getDescription = (element) => {
+  const currentTitleNodes = element
+    .querySelectorAll('.title-text');
+
+  if (!element.closest('.card-node').parentElement.closest('.card-node')) {
+    return join(currentTitleNodes);
+  }
+
+  const sectionTitleNodes = element
+    .closest('.card-node')
+    .parentElement
+    .closest('.card-node')
+    .querySelector('a:first-child .wrap-with-text')
+    .querySelectorAll('.title-text');
+
+  return `${join(sectionTitleNodes)} – ${join(currentTitleNodes)}`;
+};
+
+const getProject = (element) => {
+  return document.querySelector('.project-selector-caret').textContent;
+};
+
+togglbutton.render('.card-list-wrapper a.card', { observe: true }, (element) => {
```

Comment:
We should use a smarter selector with `:not(.toggl)` to avoid manually checking if `link` exists like in most other integrations; render will be more performant this way.

I tested with new selector `'.card-list-wrapper .card-node:not(.toggl)'`and it seems to work well (requires other changes too)*.